src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    void userNotNull_Ok() {
+        User userTest = new User();
+        userTest.setId(31654L);
+        userTest.setLogin("Bond-1234");
+        userTest.setPassword("1234567");
+        userTest.setAge(43);
+        User register = registrationService.register(userTest);
```

Comment:
You're testing the register method, but currently, you're only checking if the registered user's login is not null. Consider testing other attributes like password, ID, and age, and also ensure that the returned user is a different object from the input user.